db/schema.rb

```diff
@@ -11,12 +11,21 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20141208163419) do
+ActiveRecord::Schema.define(version: 20150306223348) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-
+  create_table "1:1", force: true do |t|
```

Comment:
a toto tu tiez nemuselo byt..ale hm neviem ako zabranit tomu aby sa to tam prestalo vytvarat